addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1843,6 +1843,7 @@ export class OdooEditor extends EventTarget {
                 title: this.options._t('Heading 1'),
                 description: this.options._t('Big section heading.'),
                 fontawesome: 'fa-header',
+                isDisabled: () => {return !this.isSelectionInBlockRoot()},
```

Comment:
I find it weird to have `() => {return ...}`. Either do:
```js
() => {
    return ...;
}
```
or, better in this case IMO:
```
() => ...,
```
There are obviously a lot of occurences of this in the diff.